canu/generate/switch/config/config.py

```diff
@@ -1129,6 +1211,15 @@ def preserve_port(
             return port["lag"]
 
 
+def get_vlan_id(destination_rack_int, sls_variables, cabinet_key):
```

Comment:
We're not great about function naming across canu code, but here it would probably be better to say what the vlan is "gotten" from.  Instead of get_vlan_id it is more clear as get_vlan_id_by_cabinet or something like that.  Ie. the function should be a bit more descriptive rather than having to read comments.